src/openrct2/drawing/Drawing.String.cpp

```diff
@@ -335,6 +335,109 @@ void draw_string_centred_raw(rct_drawpixelinfo* dpi, int32_t x, int32_t y, int32
     }
 }
 
+/**
+ * Wrap the text in buffer to width, returns width of longest line.
+ *
+ * Inserts NULL where line should break (as \n is used for something else),
+ * so the number of lines is returned in num_lines. font_height seems to be
+ * a control character for line height.
+ *
+ *  rct2: 0x006C21E2
```

Comment:
This is a brand new function. (well not really its just a subset of `gfx_wrap_string` but either way. None of the rct2 info is required.